privacyidea/static/app.js

```diff
@@ -76,6 +76,16 @@ myApp.constant("smtpServerUrl", backendUrl + instance + "/smtpserver");
 myApp.constant("radiusServerUrl", backendUrl + instance + "/radiusserver");
 myApp.constant("privacyideaServerUrl", backendUrl + instance + "/privacyideaserver");
 myApp.constant("recoveryUrl", backendUrl + instance + "/recover");
+myApp.constant("resourceNamePatterns", {
+    simple: {pattern: "^[a-zA-Z0-9_.-]+$",
+        title: "The resource name must consist of letters, numbers and '_', '-', '.'"},
```

Comment:
Please make this translateable using ``getTextCatalog``.

Reply:
fixed in 84b8a8a0